script/release.sh

```diff
@@ -134,6 +134,13 @@ git_pull() {
   cd ..
 }
 
+
+git_sync() {
+  cd $CLI
+  git pull origin
```

Comment:
I'm not sure about this :-( 
Isn't origin pointing to fork? It should pull from upstream no?